extensions/qdrant/runtime/pom.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.apache.camel.quarkus</groupId>
+		<artifactId>camel-quarkus-qdrant-parent</artifactId>
+		<version>3.9.0-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
+	<artifactId>camel-quarkus-qdrant</artifactId>
+	<name>Camel Quarkus :: Qdrant :: Runtime</name>
+	<description>Perform operations on the Qdrant Vector Database.</description>
+
+	<properties>
+		<camel.quarkus.jvmSince>3.9.0</camel.quarkus.jvmSince>
+		<camel.quarkus.nativeSince>3.9.0</camel.quarkus.nativeSince>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.camel.quarkus</groupId>
+			<artifactId>camel-quarkus-core</artifactId>
+		</dependency>
+		<dependency>
```

Comment:
@jamesnetherton @ppalaga  A dependency on camel-quarkus-grpc is needed to have the native tests run locally.
Would you have any idea to integrate this cleanly in the source tree ? We need  kind of support project ?

Reply:
Adding `quarkus-grpc-common` should be enough, no?

Reply:
Indeed, it might be sufficient. There is an issue but it's not related to qdrant :+1: